portfolio/src/main/webapp/index.html

```diff
@@ -87,11 +89,14 @@ <h2>Awards:</h2>
       </ul>
       <br>
     </div>
+
+    <h2>Places I've been to!</h2>
+    <div id="map"></div>
     
     <!--Comments Section-->
     <div>
       <h2>Comments Section</h2>
-      <p>Feel free to leave a comment here!</p>
+      <p>Feel free to leave a comment here! They are all automatically translated to Czech!</p>
```

Comment:
lol, nice. Do you also display the original language? Might be useful from user experience perspective to display both.

Reply:
Yes! The comment appears in both languages, first the original and then the Czech one